src/main/kotlin/uk/gov/justice/digital/assessments/api/AssessmentSubjectDto.kt

```diff
@@ -23,6 +23,9 @@ class AssessmentSubjectDto(
   @Schema(description = "Date of birth", example = "1929-08-08")
   val dob: LocalDate? = null,
 
+  @Schema(description = "Date of birth", example = "1929-08-08")
+  val gender: String? = null,
```

Comment:
This has been kept nullable because the subject can still be created from the court service, which does not include the gender data.